### PR TITLE
Use asset root for symbols restore

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: external/runtime/runtime.depproj [hunk 1]
```diff
@@ -82,7 +82,10 @@
 
     <ItemGroup>
       <_CoreCLRSymbolPackagesToDownload Include="@(ReferenceCopyLocalPaths->'%(NuGetPackageId)')" Condition="$([System.String]::Copy('%(Identity)').EndsWith('System.Private.CoreLib.dll'))">
-        <Url>https://dotnet.myget.org/F/dotnet-core/symbols/%(NuGetPackageId)/%(NuGetPackageVersion)/</Url>
+        <!-- If there is an asset root URL set (for orchestrated build), then append symbols to that to grab the coreclr symbol location.  The orchestrated build uses a flat package layout
+             layout. -->
```
Comment (dagood) [inline external/runtime/runtime.depproj]: \<Nit> Repeated `layout`

File: external/runtime/runtime.depproj [hunk 2]
```diff
@@ -82,7 +82,10 @@
...
+        <Url Condition="'$(DotNetAssetRootUrl)'!=''">$(AssetRootUrl)symbols/%(NuGetPackageId).%(NuGetPackageVersion).symbols.nupkg</Url>
```
Comment (dagood) [inline external/runtime/runtime.depproj]: Is `AssetRootUrl` defined somewhere or did you mean `DotNetAssetRootUrl`?

Comment (mmitche) [reply external/runtime/runtime.depproj]: Was renaming things before pushing.   Fixed.

**Comment by mmitche** [comment]
Currently in testing in the orchestrated build.  Don't merge yet.

**Comment by weshaggard** [comment]
FYI: @safern

**Pull request merged**